.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "automatic"
+}
```

Comment:
This file does not need to be added. I believe the repository does not include anything specific to Eclipse/IntelliJ/etc. If you use this file locally, you can add a global `.gitignore` to prevent Git from including it with your changes (or just ignore it when committing your changes).

Reply:
Right, It was a mistake 👍

BTW, there is a `.idea` directory pushed before, so I will remove it too as it seems a mistake because there is a `.idea` on `.gitignore`